sql/core/src/test/scala/org/apache/spark/sql/execution/SameResultSuite.scala

```diff
@@ -58,4 +61,16 @@ class SameResultSuite extends QueryTest with SharedSQLContext {
     val df4 = spark.range(10).agg(sumDistinct($"id"))
     assert(df3.queryExecution.executedPlan.sameResult(df4.queryExecution.executedPlan))
   }
+
+  test("Canonicalized result is not case-insensitive") {
```

Comment:
`Canonicalized result is not case-insensitive` -> `Canonicalized result is case-insensitive`

Reply:
Modified, thanks.